# Russian message translation file for libpq
# Copyright (C) 2001-2016 PostgreSQL Global Development Group
# This file is distributed under the same license as the PostgreSQL package.
# Serguei A. Mokhov <mokhov@cs.concordia.ca>, 2001-2004.
# Oleg Bartunov <oleg@sai.msu.su>, 2005.
# Andrey Sudnik <sudnikand@yandex.ru>, 2010.
# Alexander Lakhin <exclusion@gmail.com>, 2012-2017, 2018, 2019.
msgid ""
msgstr ""
"Project-Id-Version: libpq (PostgreSQL current)\n"
"Report-Msgid-Bugs-To: pgsql-bugs@lists.postgresql.org\n"
"POT-Creation-Date: 2020-02-05 11:03+0300\n"
"PO-Revision-Date: 2019-09-23 09:30+0300\n"
"Last-Translator: Alexander Lakhin <exclusion@gmail.com>\n"
"Language-Team: Russian <pgsql-ru-general@postgresql.org>\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: fe-auth-scram.c:183
msgid "malformed SCRAM message (empty message)\n"
msgstr "неправильное сообщение SCRAM (пустое содержимое)\n"

#: fe-auth-scram.c:189
msgid "malformed SCRAM message (length mismatch)\n"
msgstr "неправильное сообщение SCRAM (некорректная длина)\n"

#: fe-auth-scram.c:238
msgid "incorrect server signature\n"
msgstr "некорректная сигнатура сервера\n"

#: fe-auth-scram.c:247
msgid "invalid SCRAM exchange state\n"
msgstr "ошибочное состояние обмена SCRAM\n"

#: fe-auth-scram.c:270
#, c-format
msgid "malformed SCRAM message (attribute \"%c\" expected)\n"
msgstr "неправильное сообщение SCRAM (ожидался атрибут \"%c\")\n"

#: fe-auth-scram.c:279
#, c-format
msgid ""
"malformed SCRAM message (expected character \"=\" for attribute \"%c\")\n"
msgstr ""
"неправильное сообщение SCRAM (для атрибута \"%c\" ожидался символ \"=\")\n"

#: fe-auth-scram.c:320
msgid "could not generate nonce\n"
msgstr "не удалось сгенерировать разовый код\n"

#: fe-auth-scram.c:328 fe-auth-scram.c:395 fe-auth-scram.c:517
#: fe-auth-scram.c:537 fe-auth-scram.c:563 fe-auth-scram.c:577
#: fe-auth-scram.c:626 fe-auth-scram.c:660 fe-auth.c:290 fe-auth.c:360
#: fe-auth.c:395 fe-auth.c:581 fe-auth.c:740 fe-auth.c:1052 fe-auth.c:1200
#: fe-connect.c:864 fe-connect.c:1326 fe-connect.c:1502 fe-connect.c:2112
#: fe-connect.c:2135 fe-connect.c:2858 fe-connect.c:4536 fe-connect.c:4788
#: fe-connect.c:4907 fe-connect.c:5161 fe-connect.c:5241 fe-connect.c:5340
#: fe-connect.c:5596 fe-connect.c:5625 fe-connect.c:5697 fe-connect.c:5721
#: fe-connect.c:5739 fe-connect.c:5840 fe-connect.c:5849 fe-connect.c:6205
#: fe-connect.c:6355 fe-exec.c:2748 fe-exec.c:3495 fe-exec.c:3660
#: fe-gssapi-common.c:111 fe-lobj.c:895 fe-protocol2.c:1213 fe-protocol3.c:999
#: fe-protocol3.c:1703 fe-secure-common.c:110 fe-secure-gssapi.c:498
#: fe-secure-openssl.c:438 fe-secure-openssl.c:1025
msgid "out of memory\n"
msgstr "нехватка памяти\n"

#: fe-auth-scram.c:555
msgid "invalid SCRAM response (nonce mismatch)\n"
msgstr "неверный ответ SCRAM (несовпадение проверочного кода)\n"

#: fe-auth-scram.c:586
msgid "malformed SCRAM message (invalid salt)\n"
msgstr "неправильное сообщение SCRAM (некорректная соль)\n"

#: fe-auth-scram.c:600
msgid "malformed SCRAM message (invalid iteration count)\n"
msgstr "неправильное сообщение SCRAM (некорректное число итераций)\n"

#: fe-auth-scram.c:606
msgid "malformed SCRAM message (garbage at end of server-first-message)\n"
msgstr ""
"неправильное сообщение SCRAM (мусор в конце первого сообщения сервера)\n"

#: fe-auth-scram.c:637
#, c-format
msgid "error received from server in SCRAM exchange: %s\n"
msgstr "в ходе обмена SCRAM от сервера получена ошибка: %s\n"

#: fe-auth-scram.c:653
msgid "malformed SCRAM message (garbage at end of server-final-message)\n"
msgstr ""
"неправильное сообщение SCRAM (мусор в конце последнего сообщения сервера)\n"

#: fe-auth-scram.c:671
msgid "malformed SCRAM message (invalid server signature)\n"
msgstr "неправильное сообщение SCRAM (неверная сигнатура сервера)\n"

#: fe-auth.c:77
#, c-format
msgid "out of memory allocating GSSAPI buffer (%d)\n"
msgstr "недостаточно памяти для буфера GSSAPI (%d)\n"

#: fe-auth.c:132
msgid "GSSAPI continuation error"
msgstr "ошибка продолжения в GSSAPI"

#: fe-auth.c:159 fe-auth.c:389 fe-gssapi-common.c:98 fe-secure-common.c:98
msgid "host name must be specified\n"
msgstr "требуется указать имя сервера\n"

#: fe-auth.c:166
msgid "duplicate GSS authentication request\n"
msgstr "повторный запрос аутентификации GSS\n"

#: fe-auth.c:231
#, c-format
msgid "out of memory allocating SSPI buffer (%d)\n"
msgstr "недостаточно памяти для буфера SSPI (%d)\n"

#: fe-auth.c:279
msgid "SSPI continuation error"
msgstr "ошибка продолжения в SSPI"

#: fe-auth.c:350
msgid "duplicate SSPI authentication request\n"
msgstr "повторный запрос аутентификации SSPI\n"

#: fe-auth.c:375
msgid "could not acquire SSPI credentials"
msgstr "не удалось получить удостоверение SSPI"

#: fe-auth.c:429
msgid "duplicate SASL authentication request\n"
msgstr "повторный запрос аутентификации SASL\n"

#: fe-auth.c:487
msgid ""
"server offered SCRAM-SHA-256-PLUS authentication over a non-SSL connection\n"
msgstr ""
"сервер предложил аутентификацию SCRAM-SHA-256-PLUS для соединения, не "
"защищённого SSL\n"

#: fe-auth.c:499
msgid "none of the server's SASL authentication mechanisms are supported\n"
msgstr ""
"ни один из серверных механизмов аутентификации SASL не поддерживается\n"

#: fe-auth.c:605
#, c-format
msgid "out of memory allocating SASL buffer (%d)\n"
msgstr "недостаточно памяти для буфера SASL (%d)\n"

#: fe-auth.c:630
msgid ""
"AuthenticationSASLFinal received from server, but SASL authentication was "
"not completed\n"
msgstr ""
"c сервера получено сообщение AuthenticationSASLFinal, но аутентификация SASL "
"ещё не завершена\n"

#: fe-auth.c:707
msgid "SCM_CRED authentication method not supported\n"
msgstr "аутентификация SCM_CRED не поддерживается\n"

#: fe-auth.c:798
msgid "Kerberos 4 authentication not supported\n"
msgstr "аутентификация Kerberos 4 не поддерживается\n"

#: fe-auth.c:803
msgid "Kerberos 5 authentication not supported\n"
msgstr "аутентификация Kerberos 5 не поддерживается\n"

#: fe-auth.c:874
msgid "GSSAPI authentication not supported\n"
msgstr "аутентификация через GSSAPI не поддерживается\n"

#: fe-auth.c:906
msgid "SSPI authentication not supported\n"
msgstr "аутентификация через SSPI не поддерживается\n"

#: fe-auth.c:914
msgid "Crypt authentication not supported\n"
msgstr "аутентификация Crypt не поддерживается\n"

#: fe-auth.c:980
#, c-format
msgid "authentication method %u not supported\n"
msgstr "метод аутентификации %u не поддерживается\n"

#: fe-auth.c:1027
#, c-format
msgid "user name lookup failure: error code %lu\n"
msgstr "распознать имя пользователя не удалось (код ошибки: %lu)\n"

#: fe-auth.c:1037 fe-connect.c:2745
#, c-format
msgid "could not look up local user ID %d: %s\n"
msgstr "найти локального пользователя по идентификатору (%d) не удалось: %s\n"

#: fe-auth.c:1042 fe-connect.c:2750
#, c-format
msgid "local user with ID %d does not exist\n"
msgstr "локальный пользователь с ID %d не существует\n"

#: fe-auth.c:1144
msgid "unexpected shape of result set returned for SHOW\n"
msgstr "неожиданная форма набора результатов, возвращённого для SHOW\n"

#: fe-auth.c:1153
msgid "password_encryption value too long\n"
msgstr "слишком длинное значение password_encryption\n"

#: fe-auth.c:1193
#, c-format
msgid "unrecognized password encryption algorithm \"%s\"\n"
msgstr "нераспознанный алгоритм шифрования пароля \"%s\"\n"

#: fe-connect.c:1047
#, c-format
msgid "could not match %d host names to %d hostaddr values\n"
msgstr "не удалось сопоставить имена узлов (%d) со значениями hostaddr (%d)\n"

#: fe-connect.c:1123
#, c-format
msgid "could not match %d port numbers to %d hosts\n"
msgstr "не удалось сопоставить номера портов (%d) с узлами (%d)\n"

#: fe-connect.c:1219
#, c-format
msgid "invalid sslmode value: \"%s\"\n"
msgstr "неверное значение sslmode: \"%s\"\n"

#: fe-connect.c:1240
#, c-format
msgid "sslmode value \"%s\" invalid when SSL support is not compiled in\n"
msgstr "значение sslmode \"%s\" недопустимо для сборки без поддержки SSL\n"

#: fe-connect.c:1264
#, c-format
msgid "invalid gssencmode value: \"%s\"\n"
msgstr "неверное значение gssencmode: \"%s\"\n"

#: fe-connect.c:1273
#, c-format
msgid ""
"gssencmode value \"%s\" invalid when GSSAPI support is not compiled in\n"
msgstr ""
"значение gssencmode \"%s\" недопустимо для сборки без поддержки GSSAPI\n"

#: fe-connect.c:1308
#, c-format
msgid "invalid target_session_attrs value: \"%s\"\n"
msgstr "неверное значение target_session_attrs: \"%s\"\n"

#: fe-connect.c:1526
#, c-format
msgid "could not set socket to TCP no delay mode: %s\n"
msgstr "не удалось перевести сокет в режим TCP-передачи без задержки: %s\n"

#: fe-connect.c:1587
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running locally and accepting\n"
"\tconnections on Unix domain socket \"%s\"?\n"
msgstr ""
"не удалось подключиться к серверу: %s\n"
"\tОн действительно работает локально и принимает\n"
"\tсоединения через Unix-сокет \"%s\"?\n"

#: fe-connect.c:1624
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\" (%s) and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"не удалось подключиться к серверу: %s\n"
"\tОн действительно работает по адресу \"%s\" (%s)\n"
"\t и принимает TCP-соединения (порт %s)?\n"

#: fe-connect.c:1632
#, c-format
msgid ""
"could not connect to server: %s\n"
"\tIs the server running on host \"%s\" and accepting\n"
"\tTCP/IP connections on port %s?\n"
msgstr ""
"не удалось подключиться к серверу: %s\n"
"\tОн действительно работает по адресу \"%s\"\n"
"\t и принимает TCP-соединения (порт %s)?\n"

#: fe-connect.c:1702
#, c-format
msgid "invalid integer value \"%s\" for connection option \"%s\"\n"
msgstr ""
"неверное целочисленное значение \"%s\" для параметра соединения \"%s\"\n"

#: fe-connect.c:1732 fe-connect.c:1766 fe-connect.c:1801 fe-connect.c:1888
#: fe-connect.c:2535
#, c-format
msgid "setsockopt(%s) failed: %s\n"
msgstr "ошибка в setsockopt(%s): %s\n"

#: fe-connect.c:1854
#, c-format
msgid "WSAIoctl(SIO_KEEPALIVE_VALS) failed: %ui\n"
msgstr "ошибка в WSAIoctl(SIO_KEEPALIVE_VALS): %ui\n"

#: fe-connect.c:2226
msgid "invalid connection state, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения - возможно разрушение памяти\n"

#: fe-connect.c:2294
#, c-format
msgid "invalid port number: \"%s\"\n"
msgstr "неверный номер порта: \"%s\"\n"

#: fe-connect.c:2310
#, c-format
msgid "could not translate host name \"%s\" to address: %s\n"
msgstr "преобразовать имя \"%s\" в адрес не удалось: %s\n"

#: fe-connect.c:2323
#, c-format
msgid "could not parse network address \"%s\": %s\n"
msgstr "не удалось разобрать сетевой адрес \"%s\": %s\n"

#: fe-connect.c:2336
#, c-format
msgid "Unix-domain socket path \"%s\" is too long (maximum %d bytes)\n"
msgstr "длина пути Unix-сокета \"%s\" превышает предел (%d байт)\n"

#: fe-connect.c:2351
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s\n"
msgstr "преобразовать путь Unix-сокета \"%s\" в адрес не удалось: %s\n"

#: fe-connect.c:2472
#, c-format
msgid "could not create socket: %s\n"
msgstr "не удалось создать сокет: %s\n"

#: fe-connect.c:2494
#, c-format
msgid "could not set socket to nonblocking mode: %s\n"
msgstr "не удалось перевести сокет в неблокирующий режим: %s\n"

#: fe-connect.c:2504
#, c-format
msgid "could not set socket to close-on-exec mode: %s\n"
msgstr ""
"не удалось перевести сокет в режим закрытия при выполнении (close-on-exec): "
"%s\n"

#: fe-connect.c:2522
msgid "keepalives parameter must be an integer\n"
msgstr "параметр keepalives должен быть целым числом\n"

#: fe-connect.c:2662
#, c-format
msgid "could not get socket error status: %s\n"
msgstr "не удалось получить статус ошибки сокета: %s\n"

#: fe-connect.c:2690
#, c-format
msgid "could not get client address from socket: %s\n"
msgstr "не удалось получить адрес клиента из сокета: %s\n"

#: fe-connect.c:2732
msgid "requirepeer parameter is not supported on this platform\n"
msgstr "параметр requirepeer не поддерживается в этой ОС\n"

#: fe-connect.c:2735
#, c-format
msgid "could not get peer credentials: %s\n"
msgstr "не удалось получить учётные данные сервера: %s\n"

#: fe-connect.c:2758
#, c-format
msgid "requirepeer specifies \"%s\", but actual peer user name is \"%s\"\n"
msgstr ""
"requirepeer допускает подключение только к \"%s\", но сервер работает под "
"именем \"%s\"\n"

#: fe-connect.c:2793
#, c-format
msgid "could not send GSSAPI negotiation packet: %s\n"
msgstr "не удалось отправить пакет согласования GSSAPI: %s\n"

#: fe-connect.c:2805
msgid ""
"GSSAPI encryption required but was impossible (possibly no credential cache, "
"no server support, or using a local socket)\n"
msgstr ""
"затребовано шифрование GSSAPI, но это требование невыполнимо (возможно, "
"отсутствует кеш учётных данных, нет поддержки на сервере или используется "
"локальный сокет)\n"

#: fe-connect.c:2832
#, c-format
msgid "could not send SSL negotiation packet: %s\n"
msgstr "не удалось отправить пакет согласования SSL: %s\n"

#: fe-connect.c:2871
#, c-format
msgid "could not send startup packet: %s\n"
msgstr "не удалось отправить стартовый пакет: %s\n"

#: fe-connect.c:2941
msgid "server does not support SSL, but SSL was required\n"
msgstr "затребовано подключение через SSL, но сервер не поддерживает SSL\n"

#: fe-connect.c:2967
#, c-format
msgid "received invalid response to SSL negotiation: %c\n"
msgstr "получен неверный ответ при согласовании SSL: %c\n"

#: fe-connect.c:3057
msgid "server doesn't support GSSAPI encryption, but it was required\n"
msgstr "затребовано шифрование GSSAPI, но сервер его не поддерживает\n"

#: fe-connect.c:3068
#, c-format
msgid "received invalid response to GSSAPI negotiation: %c\n"
msgstr "получен неверный ответ при согласовании GSSAPI: %c\n"

#: fe-connect.c:3136 fe-connect.c:3169
#, c-format
msgid "expected authentication request from server, but received %c\n"
msgstr "ожидался запрос аутентификации от сервера, но получено: %c\n"

#: fe-connect.c:3416
msgid "unexpected message from server during startup\n"
msgstr "неожиданное сообщение от сервера в начале работы\n"

#: fe-connect.c:3643
#, c-format
msgid "could not make a writable connection to server \"%s:%s\"\n"
msgstr ""
"не удалось установить подключение для чтения/записи к серверу \"%s:%s\"\n"

#: fe-connect.c:3689
#, c-format
msgid "test \"SHOW transaction_read_only\" failed on server \"%s:%s\"\n"
msgstr ""
"проверка \"SHOW transaction_read_only\" не пройдена на сервере \"%s:%s\"\n"

#: fe-connect.c:3704
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения %d - возможно разрушение памяти\n"

#: fe-connect.c:4142 fe-connect.c:4202
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_CONNRESET event\n"
msgstr "ошибка в PGEventProc \"%s\" при обработке события PGEVT_CONNRESET\n"

#: fe-connect.c:4549
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://\n"
msgstr "некорректный адрес LDAP \"%s\": схема должна быть ldap://\n"

#: fe-connect.c:4564
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name\n"
msgstr "некорректный адрес LDAP \"%s\": отсутствует уникальное имя\n"

#: fe-connect.c:4575 fe-connect.c:4628
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute\n"
msgstr "некорректный адрес LDAP \"%s\": должен быть только один атрибут\n"

#: fe-connect.c:4585 fe-connect.c:4642
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)\n"
msgstr ""
"некорректный адрес LDAP \"%s\": не указана область поиска (base/one/sub)\n"

#: fe-connect.c:4596
#, c-format
msgid "invalid LDAP URL \"%s\": no filter\n"
msgstr "некорректный адрес LDAP \"%s\": нет фильтра\n"

#: fe-connect.c:4617
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number\n"
msgstr "некорректный адрес LDAP \"%s\": неверный номер порта\n"

#: fe-connect.c:4651
msgid "could not create LDAP structure\n"
msgstr "не удалось создать структуру LDAP\n"

#: fe-connect.c:4727
#, c-format
msgid "lookup on LDAP server failed: %s\n"
msgstr "ошибка поиска на сервере LDAP: %s\n"

#: fe-connect.c:4738
msgid "more than one entry found on LDAP lookup\n"
msgstr "при поиске LDAP найдено более одного вхождения\n"

#: fe-connect.c:4739 fe-connect.c:4751
msgid "no entry found on LDAP lookup\n"
msgstr "при поиске LDAP ничего не найдено\n"

#: fe-connect.c:4762 fe-connect.c:4775
msgid "attribute has no values on LDAP lookup\n"
msgstr "атрибут не содержит значений при поиске LDAP\n"

#: fe-connect.c:4827 fe-connect.c:4846 fe-connect.c:5379
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string\n"
msgstr "в строке соединения нет \"=\" после \"%s\"\n"

#: fe-connect.c:4919 fe-connect.c:5564 fe-connect.c:6338
#, c-format
msgid "invalid connection option \"%s\"\n"
msgstr "неверный параметр соединения \"%s\"\n"

#: fe-connect.c:4935 fe-connect.c:5428
msgid "unterminated quoted string in connection info string\n"
msgstr "в строке соединения не хватает закрывающей кавычки\n"

#: fe-connect.c:5018
#, c-format
msgid "definition of service \"%s\" not found\n"
msgstr "определение службы \"%s\" не найдено\n"

#: fe-connect.c:5041
#, c-format
msgid "service file \"%s\" not found\n"
msgstr "файл определений служб \"%s\" не найден\n"

#: fe-connect.c:5056
#, c-format
msgid "line %d too long in service file \"%s\"\n"
msgstr "слишком длинная строка (%d) в файле определений служб \"%s\"\n"

#: fe-connect.c:5129 fe-connect.c:5173
#, c-format
msgid "syntax error in service file \"%s\", line %d\n"
msgstr "синтаксическая ошибка в файле определения служб \"%s\" (строка %d)\n"

#: fe-connect.c:5140
#, c-format
msgid ""
"nested service specifications not supported in service file \"%s\", line %d\n"
msgstr ""
"рекурсивные определения служб не поддерживаются (файл определения служб \"%s"
"\", строка %d)\n"

#: fe-connect.c:5860
#, c-format
msgid "invalid URI propagated to internal parser routine: \"%s\"\n"
msgstr "во внутреннюю процедуру разбора строки передан ошибочный URI: \"%s\"\n"

#: fe-connect.c:5937
#, c-format
msgid ""
"end of string reached when looking for matching \"]\" in IPv6 host address "
"in URI: \"%s\"\n"
msgstr "URI не содержит символ \"]\" после адреса IPv6: \"%s\"\n"

#: fe-connect.c:5944
#, c-format
msgid "IPv6 host address may not be empty in URI: \"%s\"\n"
msgstr "IPv6, содержащийся в URI, не может быть пустым: \"%s\"\n"

#: fe-connect.c:5959
#, c-format
msgid ""
"unexpected character \"%c\" at position %d in URI (expected \":\" or \"/\"): "
"\"%s\"\n"
msgstr ""
"неожиданный символ \"%c\" в позиции %d в URI (ожидалось \":\" или \"/\"): "
"\"%s\"\n"

#: fe-connect.c:6088
#, c-format
msgid "extra key/value separator \"=\" in URI query parameter: \"%s\"\n"
msgstr "лишний разделитель ключа/значения \"=\" в параметрах URI: \"%s\"\n"

#: fe-connect.c:6108
#, c-format
msgid "missing key/value separator \"=\" in URI query parameter: \"%s\"\n"
msgstr "в параметрах URI не хватает разделителя ключа/значения \"=\": \"%s\"\n"

#: fe-connect.c:6159
#, c-format
msgid "invalid URI query parameter: \"%s\"\n"
msgstr "неверный параметр в URI: \"%s\"\n"

#: fe-connect.c:6233
#, c-format
msgid "invalid percent-encoded token: \"%s\"\n"
msgstr "неверный символ, закодированный с %%: \"%s\"\n"

#: fe-connect.c:6243
#, c-format
msgid "forbidden value %%00 in percent-encoded value: \"%s\"\n"
msgstr "недопустимое значение %%00 для символа, закодированного с %%: \"%s\"\n"

#: fe-connect.c:6606
msgid "connection pointer is NULL\n"
msgstr "нулевой указатель соединения\n"

#: fe-connect.c:6904
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "ПРЕДУПРЕЖДЕНИЕ: файл паролей \"%s\" - не обычный файл\n"

#: fe-connect.c:6913
#, c-format
msgid ""
"WARNING: password file \"%s\" has group or world access; permissions should "
"be u=rw (0600) or less\n"
msgstr ""
"ПРЕДУПРЕЖДЕНИЕ: к файлу паролей \"%s\" имеют доступ все или группа; права "
"должны быть u=rw (0600) или более ограниченные\n"

#: fe-connect.c:7007
#, c-format
msgid "password retrieved from file \"%s\"\n"
msgstr "пароль получен из файла \"%s\"\n"

#: fe-exec.c:445 fe-exec.c:2822
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "номер записи %d вне диапазона 0..%d"

#: fe-exec.c:506 fe-protocol2.c:502 fe-protocol2.c:537 fe-protocol2.c:1056
#: fe-protocol3.c:208 fe-protocol3.c:235 fe-protocol3.c:252 fe-protocol3.c:332
#: fe-protocol3.c:727 fe-protocol3.c:958
msgid "out of memory"
msgstr "нехватка памяти"

#: fe-exec.c:507 fe-protocol2.c:1402 fe-protocol3.c:1911
#, c-format
msgid "%s"
msgstr "%s"

#: fe-exec.c:816
msgid "write to server failed\n"
msgstr "ошибка при передаче данных серверу\n"

#: fe-exec.c:897
msgid "NOTICE"
msgstr "ЗАМЕЧАНИЕ"

#: fe-exec.c:955
msgid "PGresult cannot support more than INT_MAX tuples"
msgstr "PGresult не может вместить больше чем INT_MAX кортежей"

#: fe-exec.c:967
msgid "size_t overflow"
msgstr "переполнение size_t"

#: fe-exec.c:1244 fe-exec.c:1302 fe-exec.c:1348
msgid "command string is a null pointer\n"
msgstr "указатель на командную строку нулевой\n"

#: fe-exec.c:1308 fe-exec.c:1354 fe-exec.c:1449
msgid "number of parameters must be between 0 and 65535\n"
msgstr "число параметров должно быть от 0 до 65535\n"

#: fe-exec.c:1342 fe-exec.c:1443
msgid "statement name is a null pointer\n"
msgstr "указатель на имя оператора нулевой\n"

#: fe-exec.c:1362 fe-exec.c:1525 fe-exec.c:2234 fe-exec.c:2436
msgid "function requires at least protocol version 3.0\n"
msgstr "функция требует протокол минимум версии 3.0\n"

#: fe-exec.c:1480
msgid "no connection to the server\n"
msgstr "нет соединения с сервером\n"

#: fe-exec.c:1487
msgid "another command is already in progress\n"
msgstr "уже выполняется другая команда\n"

#: fe-exec.c:1601
msgid "length must be given for binary parameter\n"
msgstr "для двоичного параметра должна быть указана длина\n"

#: fe-exec.c:1864
#, c-format
msgid "unexpected asyncStatus: %d\n"
msgstr "неожиданный asyncStatus: %d\n"

#: fe-exec.c:1884
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_RESULTCREATE event\n"
msgstr "ошибка в PGEventProc \"%s\" при обработке события PGEVT_RESULTCREATE\n"

#: fe-exec.c:2044
msgid "COPY terminated by new PQexec"
msgstr "операция COPY прервана вызовом PQexec"

#: fe-exec.c:2052
msgid "COPY IN state must be terminated first\n"
msgstr "сначала должно завершиться состояние COPY IN\n"

#: fe-exec.c:2072
msgid "COPY OUT state must be terminated first\n"
msgstr "сначала должно завершиться состояние COPY OUT\n"

#: fe-exec.c:2080
msgid "PQexec not allowed during COPY BOTH\n"
msgstr "вызов PQexec не допускается в процессе COPY BOTH\n"

#: fe-exec.c:2326 fe-exec.c:2393 fe-exec.c:2483 fe-protocol2.c:1359
#: fe-protocol3.c:1842
msgid "no COPY in progress\n"
msgstr "операция COPY не выполняется\n"

#: fe-exec.c:2673
msgid "connection in wrong state\n"
msgstr "соединение в неправильном состоянии\n"

#: fe-exec.c:2704
msgid "invalid ExecStatusType code"
msgstr "неверный код ExecStatusType"

#: fe-exec.c:2731
msgid "PGresult is not an error result\n"
msgstr "В PGresult не передан результат ошибки\n"

#: fe-exec.c:2806 fe-exec.c:2829
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "номер столбца %d вне диапазона 0..%d"

#: fe-exec.c:2844
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "номер параметра %d вне диапазона 0..%d"

#: fe-exec.c:3154
#, c-format
msgid "could not interpret result from server: %s"
msgstr "не удалось интерпретировать ответ сервера: %s"

#: fe-exec.c:3393 fe-exec.c:3477
msgid "incomplete multibyte character\n"
msgstr "неполный многобайтный символ\n"

#: fe-gssapi-common.c:124
msgid "GSSAPI name import error"
msgstr "ошибка импорта имени в GSSAPI"

#: fe-lobj.c:154
msgid "cannot determine OID of function lo_truncate\n"
msgstr "не удалось определить OID функции lo_truncate\n"

#: fe-lobj.c:170
msgid "argument of lo_truncate exceeds integer range\n"
msgstr "аргумент lo_truncate не умещается в обычном целом\n"

#: fe-lobj.c:221
msgid "cannot determine OID of function lo_truncate64\n"
msgstr "не удалось определить OID функции lo_truncate64\n"

#: fe-lobj.c:279
msgid "argument of lo_read exceeds integer range\n"
msgstr "аргумент lo_read не умещается в обычном целом\n"

#: fe-lobj.c:334
msgid "argument of lo_write exceeds integer range\n"
msgstr "аргумент lo_write не умещается в обычном целом\n"

#: fe-lobj.c:425
msgid "cannot determine OID of function lo_lseek64\n"
msgstr "не удалось определить OID функции lo_lseek64\n"

#: fe-lobj.c:521
msgid "cannot determine OID of function lo_create\n"
msgstr "не удалось определить OID функции lo_create\n"

#: fe-lobj.c:600
msgid "cannot determine OID of function lo_tell64\n"
msgstr "не удалось определить OID функции lo_tell64\n"

#: fe-lobj.c:706 fe-lobj.c:815
#, c-format
msgid "could not open file \"%s\": %s\n"
msgstr "не удалось открыть файл \"%s\": %s\n"

#: fe-lobj.c:761
#, c-format
msgid "could not read from file \"%s\": %s\n"
msgstr "не удалось прочитать файл \"%s\": %s\n"

#: fe-lobj.c:835 fe-lobj.c:859
#, c-format
msgid "could not write to file \"%s\": %s\n"
msgstr "не удалось записать файл \"%s\": %s\n"

#: fe-lobj.c:946
msgid "query to initialize large object functions did not return data\n"
msgstr "запрос инициализации функций для больших объектов не вернул данные\n"

#: fe-lobj.c:995
msgid "cannot determine OID of function lo_open\n"
msgstr "не удалось определить OID функции lo_open\n"

#: fe-lobj.c:1002
msgid "cannot determine OID of function lo_close\n"
msgstr "не удалось определить OID функции lo_close\n"

#: fe-lobj.c:1009
msgid "cannot determine OID of function lo_creat\n"
msgstr "не удалось определить OID функции lo_creat\n"

#: fe-lobj.c:1016
msgid "cannot determine OID of function lo_unlink\n"
msgstr "не удалось определить OID функции lo_unlink\n"

#: fe-lobj.c:1023
msgid "cannot determine OID of function lo_lseek\n"
msgstr "не удалось определить OID функции lo_lseek\n"

#: fe-lobj.c:1030
msgid "cannot determine OID of function lo_tell\n"
msgstr "не удалось определить OID функции lo_tell\n"

#: fe-lobj.c:1037
msgid "cannot determine OID of function loread\n"
msgstr "не удалось определить OID функции loread\n"

#: fe-lobj.c:1044
msgid "cannot determine OID of function lowrite\n"
msgstr "не удалось определить OID функции lowrite\n"

#: fe-misc.c:290
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "функция pqGetInt не поддерживает integer размером %lu байт"

#: fe-misc.c:326
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "функция pqPutInt не поддерживает integer размером %lu байт"

#: fe-misc.c:637 fe-misc.c:859
msgid "connection not open\n"
msgstr "соединение не открыто\n"

#: fe-misc.c:807 fe-secure-openssl.c:206 fe-secure-openssl.c:314
#: fe-secure.c:268 fe-secure.c:385
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request.\n"
msgstr ""
"сервер неожиданно закрыл соединение\n"
"\tСкорее всего сервер прекратил работу из-за сбоя\n"
"\tдо или в процессе выполнения запроса.\n"

#: fe-misc.c:1046
msgid "timeout expired\n"
msgstr "тайм-аут\n"

#: fe-misc.c:1091
msgid "invalid socket\n"
msgstr "неверный сокет\n"

#: fe-misc.c:1114
#, c-format
msgid "select() failed: %s\n"
msgstr "ошибка в select(): %s\n"

#: fe-protocol2.c:90
#, c-format
msgid "invalid setenv state %c, probably indicative of memory corruption\n"
msgstr "неверное состояние setenv %c - возможно разрушение памяти\n"

#: fe-protocol2.c:389
#, c-format
msgid "invalid state %c, probably indicative of memory corruption\n"
msgstr "неверное состояние %c - возможно разрушение памяти\n"

#: fe-protocol2.c:478 fe-protocol3.c:185
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "от сервера во время простоя получено сообщение типа 0x%02x"

#: fe-protocol2.c:528
#, c-format
msgid "unexpected character %c following empty query response (\"I\" message)"
msgstr "неожиданный символ %c вслед за пустым ответом (сообщение \"I\")"

#: fe-protocol2.c:594
#, c-format
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)"
msgstr ""
"сервер отправил данные (сообщение \"D\") без предварительного описания "
"строки (сообщение \"T\")"

#: fe-protocol2.c:612
#, c-format
msgid ""
"server sent binary data (\"B\" message) without prior row description (\"T\" "
"message)"
msgstr ""
"сервер отправил двоичные данные (сообщение \"B\") без предварительного "
"описания строки (сообщение \"T\")"

#: fe-protocol2.c:632 fe-protocol3.c:411
#, c-format
msgid "unexpected response from server; first received character was \"%c\"\n"
msgstr "неожиданный ответ сервера; первый полученный символ: \"%c\"\n"

#: fe-protocol2.c:761 fe-protocol2.c:936 fe-protocol3.c:626 fe-protocol3.c:853
msgid "out of memory for query result"
msgstr "недостаточно памяти для результата запроса"

#: fe-protocol2.c:1414
#, c-format
msgid "lost synchronization with server, resetting connection"
msgstr "потеряна синхронизация с сервером; попытка восстановить соединение"

#: fe-protocol2.c:1536 fe-protocol2.c:1568 fe-protocol3.c:2099
#, c-format
msgid "protocol error: id=0x%x\n"
msgstr "ошибка протокола: id=0x%x\n"

#: fe-protocol3.c:367
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)\n"
msgstr ""
"сервер отправил данные (сообщение \"D\") без предварительного описания "
"строки (сообщение \"T\")\n"

#: fe-protocol3.c:432
#, c-format
msgid "message contents do not agree with length in message type \"%c\"\n"
msgstr "содержимое не соответствует длине в сообщении типа \"%c\"\n"

#: fe-protocol3.c:453
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d\n"
msgstr ""
"потеряна синхронизация с сервером: получено сообщение типа \"%c\", длина %d\n"

#: fe-protocol3.c:504 fe-protocol3.c:544
msgid "insufficient data in \"T\" message"
msgstr "недостаточно данных в сообщении \"T\""

#: fe-protocol3.c:577
msgid "extraneous data in \"T\" message"
msgstr "лишние данные в сообщении \"T\""

#: fe-protocol3.c:690
msgid "extraneous data in \"t\" message"
msgstr "лишние данные в сообщении \"t\""

#: fe-protocol3.c:761 fe-protocol3.c:793 fe-protocol3.c:811
msgid "insufficient data in \"D\" message"
msgstr "недостаточно данных в сообщении \"D\""

#: fe-protocol3.c:767
msgid "unexpected field count in \"D\" message"
msgstr "неверное число полей в сообщении \"D\""

#: fe-protocol3.c:820
msgid "extraneous data in \"D\" message"
msgstr "лишние данные в сообщении \"D\""

#: fe-protocol3.c:1012
msgid "no error message available\n"
msgstr "нет сообщения об ошибке\n"

#. translator: %s represents a digit string
#: fe-protocol3.c:1060 fe-protocol3.c:1079
#, c-format
msgid " at character %s"
msgstr " символ %s"

#: fe-protocol3.c:1092
#, c-format
msgid "DETAIL:  %s\n"
msgstr "ПОДРОБНОСТИ:  %s\n"

#: fe-protocol3.c:1095
#, c-format
msgid "HINT:  %s\n"
msgstr "ПОДСКАЗКА:  %s\n"

#: fe-protocol3.c:1098
#, c-format
msgid "QUERY:  %s\n"
msgstr "ЗАПРОС:  %s\n"

#: fe-protocol3.c:1105
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "КОНТЕКСТ:  %s\n"

#: fe-protocol3.c:1114
#, c-format
msgid "SCHEMA NAME:  %s\n"
msgstr "СХЕМА:  %s\n"

#: fe-protocol3.c:1118
#, c-format
msgid "TABLE NAME:  %s\n"
msgstr "ТАБЛИЦА:  %s\n"

#: fe-protocol3.c:1122
#, c-format
msgid "COLUMN NAME:  %s\n"
msgstr "СТОЛБЕЦ:  %s\n"

#: fe-protocol3.c:1126
#, c-format
msgid "DATATYPE NAME:  %s\n"
msgstr "ТИП ДАННЫХ:  %s\n"

#: fe-protocol3.c:1130
#, c-format
msgid "CONSTRAINT NAME:  %s\n"
msgstr "ОГРАНИЧЕНИЕ:  %s\n"

#: fe-protocol3.c:1142
msgid "LOCATION:  "
msgstr "ПОЛОЖЕНИЕ:  "

#: fe-protocol3.c:1144
#, c-format
msgid "%s, "
msgstr "%s, "

#: fe-protocol3.c:1146
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

#: fe-protocol3.c:1341
#, c-format
msgid "LINE %d: "
msgstr "СТРОКА %d: "

#: fe-protocol3.c:1736
msgid "PQgetline: not doing text COPY OUT\n"
msgstr "PQgetline можно вызывать только во время COPY OUT с текстом\n"

#: fe-secure-common.c:124
msgid "SSL certificate's name contains embedded null\n"
msgstr "имя в SSL-сертификате включает нулевой байт\n"

#: fe-secure-common.c:171
msgid "host name must be specified for a verified SSL connection\n"
msgstr "для проверенного SSL-соединения требуется указать имя узла\n"

#: fe-secure-common.c:196
#, c-format
msgid "server certificate for \"%s\" does not match host name \"%s\"\n"
msgstr ""
"серверный сертификат для \"%s\" не соответствует имени сервера \"%s\"\n"

#: fe-secure-common.c:202
msgid "could not get server's host name from server certificate\n"
msgstr "не удалось получить имя сервера из сертификата\n"

#: fe-secure-gssapi.c:201
msgid "GSSAPI wrap error"
msgstr "ошибка обёртывания сообщения в GSSAPI"

#: fe-secure-gssapi.c:209
msgid "outgoing GSSAPI message would not use confidentiality\n"
msgstr "исходящее сообщение GSSAPI не будет защищено\n"

#: fe-secure-gssapi.c:217
#, c-format
msgid "client tried to send oversize GSSAPI packet (%zu > %zu)\n"
msgstr "клиент попытался передать чрезмерно большой пакет GSSAPI (%zu > %zu)\n"

#: fe-secure-gssapi.c:350 fe-secure-gssapi.c:590
#, c-format
msgid "oversize GSSAPI packet sent by the server (%zu > %zu)\n"
msgstr "сервер передал чрезмерно большой пакет GSSAPI (%zu > %zu)\n"

#: fe-secure-gssapi.c:389
msgid "GSSAPI unwrap error"
msgstr "ошибка развёртывания сообщения в GSSAPI"

#: fe-secure-gssapi.c:399
msgid "incoming GSSAPI message did not use confidentiality\n"
msgstr "входящее сообщение GSSAPI не защищено\n"

#: fe-secure-gssapi.c:636
msgid "could not initiate GSSAPI security context"
msgstr "не удалось инициализировать контекст безопасности GSSAPI"

#: fe-secure-gssapi.c:666
msgid "GSSAPI size check error"
msgstr "ошибка проверки размера в GSSAPI"

#: fe-secure-gssapi.c:677
msgid "GSSAPI context establishment error"
msgstr "ошибка установления контекста в GSSAPI"

#: fe-secure-openssl.c:211 fe-secure-openssl.c:319 fe-secure-openssl.c:1219
#, c-format
msgid "SSL SYSCALL error: %s\n"
msgstr "ошибка SSL SYSCALL: %s\n"

#: fe-secure-openssl.c:218 fe-secure-openssl.c:326 fe-secure-openssl.c:1223
msgid "SSL SYSCALL error: EOF detected\n"
msgstr "ошибка SSL SYSCALL: конец файла (EOF)\n"

#: fe-secure-openssl.c:229 fe-secure-openssl.c:337 fe-secure-openssl.c:1232
#, c-format
msgid "SSL error: %s\n"
msgstr "ошибка SSL: %s\n"

#: fe-secure-openssl.c:244 fe-secure-openssl.c:352
msgid "SSL connection has been closed unexpectedly\n"
msgstr "SSL-соединение было неожиданно закрыто\n"

#: fe-secure-openssl.c:250 fe-secure-openssl.c:358 fe-secure-openssl.c:1241
#, c-format
msgid "unrecognized SSL error code: %d\n"
msgstr "нераспознанный код ошибки SSL: %d\n"

#: fe-secure-openssl.c:398
msgid "could not determine server certificate signature algorithm\n"
msgstr "не удалось определить алгоритм подписи сертификата сервера\n"

#: fe-secure-openssl.c:419
#, c-format
msgid "could not find digest for NID %s\n"
msgstr "не удалось найти алгоритм хеширования по NID %s\n"

#: fe-secure-openssl.c:429
msgid "could not generate peer certificate hash\n"
msgstr "не удалось сгенерировать хеш сертификата сервера\n"

#: fe-secure-openssl.c:486
msgid "SSL certificate's name entry is missing\n"
msgstr "запись имени в SSL-сертификате отсутствует\n"

#: fe-secure-openssl.c:815
#, c-format
msgid "could not create SSL context: %s\n"
msgstr "не удалось создать контекст SSL: %s\n"

#: fe-secure-openssl.c:852
#, c-format
msgid "could not read root certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл корневых сертификатов \"%s\": %s\n"

#: fe-secure-openssl.c:880
#, c-format
msgid "SSL library does not support CRL certificates (file \"%s\")\n"
msgstr "Библиотека SSL не поддерживает проверку CRL (файл \"%s\")\n"

#: fe-secure-openssl.c:908
msgid ""
"could not get home directory to locate root certificate file\n"
"Either provide the file or change sslmode to disable server certificate "
"verification.\n"
msgstr ""
"не удалось получить домашний каталог для поиска файла корневых сертификатов\n"
"Укажите полный путь к файлу или отключите проверку сертификата сервера, "
"изменив sslmode.\n"

#: fe-secure-openssl.c:912
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file or change sslmode to disable server certificate "
"verification.\n"
msgstr ""
"файл корневых сертификатов \"%s\" не существует\n"
"Укажите полный путь к файлу или отключите проверку сертификата сервера, "
"изменив sslmode.\n"

#: fe-secure-openssl.c:943
#, c-format
msgid "could not open certificate file \"%s\": %s\n"
msgstr "не удалось открыть файл сертификата \"%s\": %s\n"

#: fe-secure-openssl.c:962
#, c-format
msgid "could not read certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл сертификата \"%s\": %s\n"

#: fe-secure-openssl.c:987
#, c-format
msgid "could not establish SSL connection: %s\n"
msgstr "не удалось установить SSL-соединение: %s\n"

#: fe-secure-openssl.c:1041
#, c-format
msgid "could not load SSL engine \"%s\": %s\n"
msgstr "не удалось загрузить модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure-openssl.c:1053
#, c-format
msgid "could not initialize SSL engine \"%s\": %s\n"
msgstr "не удалось инициализировать модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure-openssl.c:1069
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось прочитать закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"

#: fe-secure-openssl.c:1083
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось загрузить закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"

#: fe-secure-openssl.c:1120
#, c-format
msgid "certificate present, but not private key file \"%s\"\n"
msgstr "сертификат присутствует, но файла закрытого ключа \"%s\" нет\n"

#: fe-secure-openssl.c:1128
#, c-format
msgid ""
"private key file \"%s\" has group or world access; permissions should be "
"u=rw (0600) or less\n"
msgstr ""
"к файлу закрытого ключа \"%s\" имеют доступ все или группа; права должны "
"быть u=rw (0600) или более ограниченные\n"

#: fe-secure-openssl.c:1139
#, c-format
msgid "could not load private key file \"%s\": %s\n"
msgstr "не удалось загрузить файл закрытого ключа \"%s\": %s\n"

#: fe-secure-openssl.c:1153
#, c-format
msgid "certificate does not match private key file \"%s\": %s\n"
msgstr "сертификат не соответствует файлу закрытого ключа \"%s\": %s\n"

#: fe-secure-openssl.c:1262
#, c-format
msgid "certificate could not be obtained: %s\n"
msgstr "не удалось получить сертификат: %s\n"

#: fe-secure-openssl.c:1351
#, c-format
msgid "no SSL error reported"
msgstr "нет сообщения об ошибке SSL"

#: fe-secure-openssl.c:1360
#, c-format
msgid "SSL error code %lu"
msgstr "код ошибки SSL: %lu"

#: fe-secure.c:276
#, c-format
msgid "could not receive data from server: %s\n"
msgstr "не удалось получить данные с сервера: %s\n"

#: fe-secure.c:392
#, c-format
msgid "could not send data to server: %s\n"
msgstr "не удалось передать данные серверу: %s\n"

#: win32.c:317
#, c-format
msgid "unrecognized socket error: 0x%08X/%d"
msgstr "нераспознанная ошибка сокета: 0x%08X/%d"

#~ msgid "could not get home directory to locate password file\n"
#~ msgstr "не удалось получить домашний каталог для загрузки файла паролей\n"

#~ msgid "could not get home directory to locate service definition file"
#~ msgstr ""
#~ "не удалось получить домашний каталог для загрузки файла определений служб"

#~ msgid "setsockopt(TCP_KEEPIDLE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPIDLE): %s\n"

#~ msgid "setsockopt(TCP_KEEPALIVE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPALIVE): %s\n"

#~ msgid "setsockopt(TCP_KEEPINTVL) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPINTVL): %s\n"

#~ msgid "setsockopt(SO_KEEPALIVE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(SO_KEEPALIVE): %s\n"

#~ msgid "could not acquire mutex: %s\n"
#~ msgstr "не удалось заблокировать семафор: %s\n"

#~ msgid "socket not open\n"
#~ msgstr "сокет не открыт\n"

#~ msgid "unrecognized return value from row processor"
#~ msgstr "процессор строк вернул нераспознанное значение"

#~ msgid "could not restore nonblocking mode on socket: %s\n"
#~ msgstr "не удалось вернуть сокет в неблокирующий режим: %s\n"

#~ msgid "Kerberos 5 authentication rejected: %*s\n"
#~ msgstr "аутентификация Kerberos 5 не пройдена: %*s\n"

#~ msgid "could not set socket to blocking mode: %s\n"
#~ msgstr "не удалось перевести сокет в блокирующий режим: %s\n"
